Changelog.md

```diff
@@ -1,6 +1,8 @@
 ### 0.8.16 (unreleased)
 
 Language Features:
+ * Array literals are convertible to both dynamically-sized and statically-sized arrays.
+ * String literals being part of an array literal can be implicitly converted into bytes.
```

Comment:
Do we really want to allow this? Is it necessary?

Reply:
This is not a must. We can block that conversion. But, I think it is rather useful and makes compiler more predictive. I mean, why to prevent `bytes[2] memory st = ["foo", "bar"];` while `bytes memory s = "foo";` is perfectly fine?